_blog/2018/aws-nlb.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Configure Istio Ingress with AWS NLB"
+overview: Describes how to configure Istio ingress with an external load balancer Network Load Balancer on AWS
+publish_date: April 10, 2018
+subtitle: Ingress AWS Network Load Balancer
+attribution: Julien SENON
+
+order: 90
+
+layout: blog
+type: markdown
+redirect_from: "/blog/aws-nlb.html"
+---
+{% include home.html %}
+
+Network load balancer (NLB) could be used instead of classical load balancer. You can find comparison between different AWS `loadbalancer` [here](https://aws.amazon.com/elasticloadbalancing/details/#compare).
+
+This blog entry will provide instructions to use and configure ingress Istio with [AWS Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html).
```

Comment:
I think it would read better if this paragraph came first. Start by telling what this blog is about, and _then_ mention that the NLB can be used instead of the classic LB.